liveblog.php

```diff
@@ -1068,6 +1069,8 @@ public static function enqueue_scripts() {
 						'class_term_prefix'            => __( 'term-', 'liveblog' ),
 						'class_alert'                  => __( 'type-alert', 'liveblog' ),
 						'class_key'                    => __( 'type-key', 'liveblog' ),
+
+						'author_list_debounce_time'    => apply_filters( 'liveblog_author_list_debounce_time', self::AUTHOR_LIST_DEBOUNCE_TIME ),
```

Comment:
Filter should be [documented](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters).

(I know most of the others aren't yet, but it's got to start somewhere, and new code is a good a place as any!)